### PR TITLE
aws-kafka: add page

### PR DESCRIPTION
**Pull request by giladAlboher**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/aws-kafka.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# AWS Kafka CLI Commands
+
+> AWS CLI commands for managing Amazon Managed Streaming for Apache Kafka (Amazon MSK).
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/kafka/index.html>.
+
+- Create a Kafka Cluster:
+
+`aws kafka create-cluster --cluster-name <cluster-name> --broker-node-group-info instanceType=<instance-type>,clientSubnets=<subnet-ids> --kafka-version <kafka-version> --number-of-broker-nodes <number-of-broker-nodes>`
```
Comment (mfrw) [inline pages/common/aws-kafka.md:8]:
```suggestion
`aws kafka create-cluster --cluster-name {{cluster_name}} --broker-node-group-info instanceType={{instance_type}},clientSubnets={{subnet_ids}} --kafka-version {{kafka_version}} --number-of-broker-nodes {{number_of_broker_nodes}}`
```

File: pages/common/aws-kafka.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Describe a Kafka Cluster:
+
+`aws kafka describe-cluster --cluster-arn <cluster-arn>`
```
Comment (mfrw) [inline pages/common/aws-kafka.md:12]:
```suggestion
`aws kafka describe-cluster --cluster-arn {{cluster_arn}}`
```

File: pages/common/aws-kafka.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- List Kafka Clusters:
+
+`aws kafka list-clusters`
+
+- Create a Configuration:
+
+`aws kafka create-configuration --name <configuration-name> --server-properties file://<path-to-server-properties-file>`
```
Comment (mfrw) [inline pages/common/aws-kafka.md:20]:
```suggestion
`aws kafka create-configuration --name {{configuration_name}} --server-properties {{file://{{path_to_server_properties_file}}`
```

File: pages/common/aws-kafka.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Describe Configuration:
+
+`aws kafka describe-configuration --arn <configuration-arn>`
```
Comment (mfrw) [inline pages/common/aws-kafka.md:24]:
```suggestion
`aws kafka describe-configuration --arn {{configuration_arn}}`
```

File: pages/common/aws-kafka.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- List Configurations:
+
+`aws kafka list-configurations`
+
+- Update a Kafka Cluster Configuration:
+
+`aws kafka update-cluster-configuration --cluster-arn <cluster-arn> --configuration-info arn=<configuration-arn>,revision=<configuration-revision>`
```
Comment (mfrw) [inline pages/common/aws-kafka.md:32]:
```suggestion
`aws kafka update-cluster-configuration --cluster-arn {{cluster_arn}} --configuration-info arn={{configuration_arn}},revision={{configuration_revision}}`
```

File: pages/common/aws-kafka.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Delete a Kafka Cluster:
+
+`aws kafka delete-cluster --cluster-arn <cluster-arn>`
```
Comment (mfrw) [inline pages/common/aws-kafka.md:36]:
```suggestion
`aws kafka delete-cluster --cluster-arn {{cluster_arn}}`
```

**Review by mfrw — COMMENTED** [review]
Thank you @giladAlboher for your contribution. I have a few review comments.
Place holders should be enclsed within `{{ }}` instead of `< >`
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md

File: pages/common/aws-kafka.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws kafka list-clusters`
+
+- Create a Configuration:
+
+`aws kafka create-configuration --name {{configuration_name}} --server-properties {{file://{{path_to_server_properties_file}}`
+
+- Describe Configuration:
+
+`aws kafka describe-configuration --arn {{configuration_arn}}`
```
Comment (kbdharun) [inline pages/common/aws-kafka.md:24]:
```suggestion
- Describe a MSK configuration:

`aws kafka describe-configuration --arn {{configuration_arn}}`
```

File: pages/common/aws-kafka.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- List Configurations:
```
Comment (kbdharun) [inline pages/common/aws-kafka.md:26]:
```suggestion
- List all MSK configurations in the current region:
```

File: pages/common/aws-kafka.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws kafka list-configurations`
+
+- Update a Kafka Cluster Configuration:
```
Comment (kbdharun) [inline pages/common/aws-kafka.md:30]:
```suggestion
- Update the MSK cluster configuration:
```

File: pages/common/aws-kafka.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws kafka update-cluster-configuration --cluster-arn {{cluster_arn}} --configuration-info arn={{configuration_arn}},revision={{configuration_revision}}`
+
+- Delete a Kafka Cluster:
```
Comment (kbdharun) [inline pages/common/aws-kafka.md:34]:
```suggestion
- Delete the MSK cluster:
```

File: pages/common/aws-kafka.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
+# AWS Kafka CLI Commands
```
Comment (kbdharun) [inline pages/common/aws-kafka.md:1]:
```suggestion
# aws kafka
```
The page title must match with the command name.

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution. I have some suggestions for this page.

---

_A minor nitpick from my side, for descriptions I wouldn't suggest capitalizing successive terms unless it's required. i.e. Use `Create a new MSK cluster` instead of `Create a New MSK Cluster`._

File: pages/common/aws-kafka.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws kafka describe-cluster --cluster-arn {{cluster_arn}}`
+
+- List Kafka Clusters:
+
+`aws kafka list-clusters`
+
+- Create a Configuration:
+
+`aws kafka create-configuration --name {{configuration_name}} --server-properties {{file://{{path_to_server_properties_file}}`
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages/common/aws-kafka.md:21]: @giladAlboher With this suggestion addressed, I think the PR is GTG.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**